.tool-versions

```diff
@@ -1 +1 @@
-nodejs 16.13.0
+nodejs 16.14.2
```

Comment:
This doesn't need to be handled now, but shouldn't `.tool-versions` be git ignored?

Reply:
Let's focus on getting this PR out, but would definitely take a note to fix this later once header is in, if it should be ignored.

Reply:
it could be, but i thought we agreed on adding it to the repo, we can remove it in a later PR if we decide on ignoring